package.json

```diff
@@ -4,14 +4,17 @@
   "dependencies": {
     "babel-core": "^6.23.0",
     "babel-loader": "^6.4.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.2.1",
     "babel-preset-react": "^6.24.1",
     "backbone": "~1.3.2",
     "backbone.paginator": "~2.0.3",
     "coffee-loader": "^0.7.3",
     "coffee-script": "1.6.1",
-    "@edx/studio-frontend": "0.1.0",
-    "edx-bootstrap": "^0.2.1",
+    "css-loader": "^0.28.5",
+    "@edx/edx-bootstrap": "0.3.3",
+    "@edx/studio-frontend": "edx/studio-frontend#27668cd9595eb1f6a6e68fcab3d3ac5273aec50a",
```

Comment:
Nit-question: I know the general pattern is to alphabetize these listings; should that guideline be so strict as to put `@edx/studio-frontend` ahead of `edx-pattern-library`, or do we want to group all the `@edx` entries together before the plain `edx` ones?